changelog.txt

```diff
@@ -26,8 +26,11 @@ Changed
 - ``CPUEnvironment`` and ``GPUEnvironment`` classes are deprecated (#381).
 - Docstrings are now written in `numpydoc style <https://numpydoc.readthedocs.io/en/latest/format.html>`__ (#392).
 - Default environment for the University of Minnesota Mangi cluster changed from SLURM to Torque (#393).
-- Improved internal caching of scheduler status (#410).
 - Deprecated method ``export_job_statuses`` (#402).
+- Improved internal caching of scheduler status (#410).
+- ``Scheduler`` is now an abstract base class (#426).
+- ``flow.scheduling.fakescheduler`` has been renamed to ``flow.scheduling.fake_scheduler`` (#426).
+- Arguments to ``submit`` have been changed for all scheduler classes (#426).
```

Comment:
should we mention that we no longer positional arguments in this method except script.

Reply:
I didn't want to make the message any longer, and there are other specific changes that I don't want to describe here (re-ordering keyword arguments). I think this is probably okay.